source/api/project_api.ts

```diff
@@ -91,6 +92,18 @@ export default class ProjectApi {
     return Http.get(url, user) as unknown as Promise<ApiCloudFile[]>;
   }
 
+  static listProjectFiles({projectId}: AssociationIds, purposeType: ApiPurposeType | PurposeType, user: User): Promise<ApiCloudFile[]> {
```

Comment:
The method in file_information_api.ts should probably be deprecated since it looks like it's being moved here